virttest/libvirt_xml/vm_xml.py

```diff
@@ -2251,9 +2254,12 @@ def __init__(self, virsh_instance=base.virsh):
         accessors.XMLElementList('iothreadpins', self, parent_xpath='/',
                                  marshal_from=self.marshal_from_iothreadpins,
                                  marshal_to=self.marshal_to_iothreadpins)
+        accessors.XMLElementList('iothreadscheds', self, parent_xpath='/',
```

Comment:
I assume accessors.XMLElementDict can satisfy you, don't need use marchal functions

Reply:
iothreadsched attributes can be several values, I prefer list of dict here.